Octokit.Tests/Clients/RepositoryPagesClientTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using NSubstitute;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    public class RepositoryPagesClientTests
+    {
+        public class TheGetMethod
+        {
+            [Fact]
+            public void RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new RepositoryPagesClient(connection);
+
+                client.Get("fake", "repo");
+
+                connection.Received().Get<Page>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/pages"), null);
```

Comment:
Nah, I'll merge that first pass at the documentation into `master` - so the fix will land there soon.